api/custom_auth/views.py

```diff
@@ -21,3 +22,8 @@ class CustomAuthTokenLoginWithMFACode(AuthTokenLoginWithMFACode):
 
     throttle_classes = [ScopedRateThrottle]
     throttle_scope = "mfa_code"
+
+
+class ThrottledUserViewSet(UserViewSet):
+    throttle_classes = [ScopedRateThrottle]
```

Comment:
This will set the throttle for all the endpoints in this viewset whereas I think we only really want it on signup, right? Should we just override the `get_throttles` method instead?